### PR TITLE
add new OBSOLETE term MONDO:0100136 Fanconia anemia complementation group M

### LINKED ISSUE #1636: [NTR] Fanconia anemia complementation group M
Fanconia anemia complementation group M
(this might be present but I can't see all search results in the drop down/ Also tried OLS)

Child of Fanconi anemia (MONDO:0019391)

def

Any Fanconi anemia in which the cause of the disease is a mutation in the FANCM  gene.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1636

### CODE REVIEW

**Comment by cmungall** [comment]
I'm closing unless Maureen thinks it should be added in which case re-open!

**Comment by cmungall** [comment]
on second thoughts, I think born obsolete is a better path

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -433115,6 +433115,14 @@ xref: ICD10:F16.183 {source="MONDO:relatedTo"}
 is_a: MONDO:0024417 {source="https://orcid.org/0000-0001-5208-3432"} ! perceptual disorders
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:433118]: looks like you did not branch from master

**Comment by cmungall** [comment]
conflicts

**Comment by nicolevasilevsky** [comment]
fixed conflict

**Pull request merged**